### DISCUSSION TITLE
Respect user decimals settings

### DISCUSSION THREAD

**Discussion by hackhowtofaq** [grafana/grafana#38135] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
According to this https://github.com/grafana/grafana/issues/10507 
> Grafana will switch to scientific notation if there are not significant digits beyond 7 decimals
even when I set the decimals to 8

I would like to set the decimals and keep them like that without auto-switching to a grafana default.

**Why is this needed**:
It creates unnecessary cognitive fatigue, if you are asking me, and it is not respecting the Principle of least surprise.
![image](https://user-images.githubusercontent.com/1138336/89864138-19b35200-dbb4-11ea-8ee5-c397d53982c3.png)
![image](https://user-images.githubusercontent.com/1138336/89901414-b7774300-dbed-11ea-8619-6b627f2bf036.png)

Thank you.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
